components/k8s-model-server/docker/Dockerfile.gpu

```diff
@@ -0,0 +1,96 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM nvidia/cuda:8.0-cudnn6-devel-ubuntu16.04
+
+MAINTAINER Jingtian Peng <pjt73651@gmail.com>
+
+RUN apt-get update && apt-get install -y \
+        build-essential \
+        curl \
+        libcurl3-dev \
+        git \
+        libfreetype6-dev \
+        libpng12-dev \
+        libzmq3-dev \
+        pkg-config \
+        python-dev \
+        python-numpy \
+        python-pip \
+        software-properties-common \
+        swig \
+        zip \
+        zlib1g-dev \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Set up grpc
+RUN pip install mock grpcio
+
+# Set up all envs
+ENV BAZELRC=/root/.bazelrc \
```

Comment:
https://github.com/tensorflow/tensorflow/blob/r1.4/tensorflow/tools/docker/Dockerfile.devel-gpu#L80-L83
certain env are already default  though...

Reply:
What's the meaning of `already default`? There is no exported env in the Dockerfile.gpu

Reply:
oh... sorry, by that i mean, you are just setting the default value... which can be omitted following the tensorflow dockerfile example..

Reply:
I see. How about replacing `env` with `args` ?

Reply:
sounds good!